pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -271,4 +287,32 @@ pub mod pallet {
 			Ok(())
 		}
 	}
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
+		fn on_initialize(block_number: T::BlockNumber) -> Weight {
+			let active_season_id = Self::active_season_id();
+			let next_season_id = Self::next_active_season_id();
+			let season_id = active_season_id.unwrap_or(next_season_id);
+			let maybe_season = Self::seasons(season_id);
```

Comment:
Let's remove some variables and add `db_weight` here:
```suggestion
			let season_id = Self::active_season_id().unwrap_or(Self::next_active_season_id());
			let mut db_weight = T::DbWeight::get().reads(2);
```

Reply:
done